### PR TITLE
Height

### PR DESCRIPTION
**Pull request by TPANCHULIDZE**

### CODE REVIEW

File: ideal_weight.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Human
+	def ideal_weight(name, height)
+		if height - 110 >= 0
+			return "Hello #{name}, your height is #{height} and your optimal weight is #{height - 110}"
```
Comment (alexshvedko) [inline ideal_weight.rb]: why did you add `return`?

Comment (alexshvedko) [reply ideal_weight.rb]: align code

File: ideal_weight.rb [hunk 2]
```diff
@@ -1,18 +1,28 @@
-class Human
-	def ideal_weight(name, height)
-		if height - 110 >= 0
-			return "Hello #{name}, your height is #{height} and your optimal weight is #{height - 110}"
-		else
-			return "Hello #{name}, Your weight is already optimal"
-		end
-	end
+# calculate ideal weight
```
Comment (TPANCHULIDZE) [inline ideal_weight.rb:1]: recorrect code
